src/pages/profile/components/input/index.ts

```diff
@@ -1,8 +1,16 @@
 import template from './input.hbs';
-import Component from '../../../../utils/Component';
+import { TextInput, TextInputProps } from '../../../../components/text-input';
+
+export type ProfileInputProps = { disabled: boolean } & TextInputProps;
+export class ProfileInput extends TextInput {
+  constructor(propsAndChildren: any) {
+    super({ ...propsAndChildren });
```

Comment:
Копирование здесь излишне
```suggestion
    super(propsAndChildren);
```